examples/src/main/java/com/mapbox/navigation/examples/core/MapboxNavigationActivity.kt

```diff
@@ -293,7 +293,8 @@ class MapboxNavigationActivity : AppCompatActivity() {
                 .accessToken(getMapboxAccessTokenFromResources())
                 .eventsAppMetadata(
                     EventsAppMetadata.Builder(
-                        BuildConfig.APPLICATION_ID, BuildConfig.VERSION_NAME
+                        BuildConfig.APPLICATION_ID,
+                        BuildConfig.VERSION_NAME
```

Comment:
Why are forced to put each parameter on a separate line now? According to Kotlin [Coding conventions](https://kotlinlang.org/docs/coding-conventions.html#method-calls) this is allowed:
> Group multiple closely related arguments on the same line.

Reply:
Suppose, it also depends which rules' source to use. [Google recommends to move every new element to the new line](https://developer.android.com/kotlin/style-guide#functions)

[Ktlint](https://pinterest.github.io/ktlint/) uses both sources (kotlinlang and google)
`ktlint aims to capture the [official Kotlin coding conventions](https://kotlinlang.org/docs/reference/coding-conventions.html) and [Android Kotlin Style Guide](https://android.github.io/kotlin-guides/style.html). In some aspects ktlint is a bit more strict[*](https://github.com/pinterest/ktlint/issues/284#issuecomment-425177186).`

Not sure which one KtLint use when they contradict each other